tests/BigInt.test.ts

```diff
@@ -120,9 +120,9 @@ describe('BigInt', () => {
     );
     expect(errors).toEqual(undefined);
     expect(data).toEqual({
-      a: { result: 2147483647n },
-      b: { result: 9007199254740991n },
-      d: { result: 2n },
+      a: { result: '2147483647' },
```

Comment:
Rather than implicitly intending to lose precision, it's better to let clients handle string correctly. Consider the case of using a long int as an id field. Even considering the JS clients, they may prefer string over numbers with lost precision as the `BigInt()` argument.

In the case of protobufjs, they already have the correct cast for the `typeof value ==='number'` and `typeof value ==='string'` cases.

But since they doesn't use the bigint primitive yet, it will be fallback empty (`{ low: 0, high: 0, unsigned: false }`) value.

(This problem is already being reproduced in the official example of graphql-mesh, and we should fix this in graphql-mesh anyway. Need a patch for graphql-scalars or protobufjs I think)

This is not graphql related and is a specific client issue, but considering that bigint hasn't been used in JavaScript widely, it's potentially more problematic than string.

Reply:
Also, the patch would be avoided some cases. it makes the whole library incompatible in sandboxing environments or workers where global scope cannot be modified.